tests/python/unit/test_common.py

```diff
@@ -0,0 +1,9 @@
+import pytest
```

Comment:
What is the use of this file?

Reply:
Tradition, Sir.
But ok then, removed in [80e9067](https://github.com/dataiku/dss-plugin-googlesheets/pull/4/commits/80e90679db6a7bffa644251884225fc6cce31628)